Documentation/Makefile

```diff
@@ -51,12 +51,12 @@ READTHEDOCS_VERSION:=$(READTHEDOCS_VERSION)
 DOCKER_CTR_ROOT_DIR := /src
 DOCKER_CTR_BASE := $(CONTAINER_ENGINE) container run --rm \
 		--workdir $(DOCKER_CTR_ROOT_DIR)/Documentation \
-		--volume $(CURDIR)/..:$(DOCKER_CTR_ROOT_DIR)
+		--volume $(CURDIR)/..:$(DOCKER_CTR_ROOT_DIR) \
+		--user "$(shell id -u):$(shell id -g)"
```

Comment:
I've always been wondering why we're forking two shells for this instead of using just `--user "$UID:$GID"`, do you know, by any chance? Are these variables unavailable on macOS or some specific environments?

Reply:
I'm not sure. It's common to suggest using the `id` command to gather these. I don't have a mac environment to test this out in.